app/assets/javascripts/components/student/student_comment.es6.jsx

```diff
@@ -0,0 +1,45 @@
+class StudentComment extends Component {
+
+  static get PropTypes() {
+    return {
+      comment: React.PropTypes.object.isRequired,
+    };
+  }
+
+  static get DefaultProps() {
+    return {
+      comment: {},
+    };
+  }
+
+  get clickableStyles() {
+    return {
+      default: {
+        flex: '1',
+        padding: '12px',
+        marginTop: '12px',
+        marginRight: '12px',
+        borderRadius: '1px',
+        boxSizing: 'border-box',
+      },
+      hover: {
+        backgroundColor: StyleConstants.colors.turquoise,
+      },
+    };
+  }
+
+  renderComment() {
+    return (
+      `${this.props.comment.content} - ${this.props.comment.user}`
```

Comment:
Since this isn't actually rendering a DOM element, let's just put it directly in the `Clickable` props for now. We'll generally try to reserve `render` methods for DOM elements!